requirements.txt

```diff
@@ -1,2 +1,4 @@
 pcaspy
 six
+zmq
```

Comment:
Wouldn't it be better if this was `pyzmq`?

`zmq` appears to be an alias to it, but always reports version "0.0.0".